lib/wallet/wallet.js

```diff
@@ -4284,6 +4294,110 @@ class Wallet extends EventEmitter {
     }
   }
 
+  /**
+   * Create and finalize a finalize batch
+   * without a lock.
+   * @param {String[]} names
+   * @param {Object} options
+   * @returns {Promise<BatchResponse>}
+   */
+
+  async _createBatchFinalize(names, options) {
+    const acct = options ? options.account : null;
```

Comment:
If `options` is an optional argument, the JSDoc should reflect that.
Same for the other methods